src/vm/types/signatures.rs

```diff
@@ -71,7 +71,7 @@ pub struct TupleTypeSignature {
 pub struct BufferLength(pub u32);
 
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
-pub struct StringUTF8Length(u32);
+pub struct StringUTF8Length(pub u32);
```

Comment:
This goes in the opposite direction of  https://github.com/blockstack/stacks-blockchain/pull/2757

I guess he merged this into `develop`, so the example he set is not present on the line 71 of this file.

Basically, don't make this public, and use a constructor that will check the length is not outside of the bounds..

https://github.com/blockstack/stacks-blockchain/blob/986a210ba63db49d838d73ce916b86b7bc6b9958/src/vm/types/signatures.rs#L266